x-pack/legacy/plugins/monitoring/public/components/kibana/instances/instances.js

```diff
@@ -41,7 +41,14 @@ const getColumns = (kbnUrl, scope) => {
       field: 'status',
       render: (status, kibana) => (
         <div
-          title={`Instance status: ${status}`}
+          title={
+            i18n.translate('xpack.monitoring.kibana.listing.instanceStatusTitle', {
+              defaultMessage: 'Instance status: {kibanaStatus}',
+              values: {
+                kibanaStatus: status
+              }
+            })
+          }
```

Comment:
Can you move this change to it's own PR? Let's keep the scope of this PR to just removing dead code. Atomic PRs are good for several reasons, including making them easier to review + allowing us to selectively backport certain changes vs. others.

Reply:
@ycombinator I agree 💯 (and was thinking the same), but if this is moved to a different PR then this PR will fail `node scripts/i18n_check` unless I explicitly remove all `instanceStatusTitle` translations and add them back with the other PR. This becomes more difficult as all i18n translations are subjected to different review/contribution standards

Let me know if you still want me to go this direction though

Reply:
I see. I don't know the codebase too well but that sounds like a lot of work, and this particular bit of extraneous code is not too much. So I'd say keep things as is.

Thanks for the explanation!